### DISCUSSION TITLE
リンクに:visitedをつけてほしい

### DISCUSSION THREAD

**Discussion by NurseAngel** [increments/qiita-discussions#90] [要望・アイデア・不具合]
記事が訪問済か否かを区別できません。

予定がないのであれば、せめてセマンティックなclass名を付けてほしいです。

```
.css-81mxb5 a:visited {
    color: #9f9db9;
}
```

これまでこれを使っていたのですが、サイトトップで効かなくなりました。
変更後。

```
.css-81mxb5 a:visited {
    color: #9f9db9;
}
.css-18gxgni a:visited {
    color: #9f9db9;
}
```

class名の意味がわからない。

**Comment by kskwtnk** [comment]
@NurseAngel 

ご意見ありがとうございます！

`:visited`のスタイル、付与します 🙇 

---

クラス名ですがQiitaではCSS in JSのライブラリとして[Emotion](https://emotion.sh/docs/introduction)を使っていて、最終的に吐き出すクラス名にはハッシュ値が付与されることになっています
そのためスタイルが変わるとハッシュも変わり、クラス名に意味性を持たせない運用をしています 🙏

**Comment by kskwtnk** [comment]
@NurseAngel 
こちら、スタイルを付与してリリースしました！
